python/tests/stress/arcticdb/version_store/test_mem_leaks.py

```diff
@@ -723,7 +750,106 @@ def test_mem_leak_read_all_arctic_lib_memray(library_with_big_symbol_):
         logger.info("Test starting")
         st = time.time()
         data: pd.DataFrame = lib.read(symbol).data
+        lib.head
         del data
         logger.info(f"Test took : {time.time() - st}")
 
         gc.collect()
+
+    @pytest.fixture
+    def prepare_head_tails_symbol(lmdb_library_any):
+        """
+        This fixture is part of test `test_mem_leak_head_tail_memray`
+
+        It creates a symbol with several versions and snapshot for each version.
+        It inserts dataframes which are large given the segment size of library.
+        And if the dynamic schema is used each version has more columns than previous version
+
+        Should not be reused
+        """
+        lib: Library = lmdb_library_any
+        opts: LibraryOptions = lib.options()
+        total_number_columns = 1002
```

Comment:
What column and row slicing options are we using here? It's not very clear from reading the test